cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -98,43 +117,101 @@ started correctly.
+
+    $ docker run -it -p 11015:11015 -p 11011:11011 --name cdap-sdk caskdata/cdap-standalone cdap.sh start
+    
+- Starting the Standalone CDAP, in the foreground, with ports forwarded, and with debugging enabled::
+
+    $ docker run -it -p 11015:11015 -p 11011:11011 --name cdap-sdk caskdata/cdap-standalone cdap.sh start --enable-debug
+
+- For information on mounting volumes and sharing data with the container, see the
+  examples in Docker's documentation on `data volumes
+  <https://docs.docker.com/engine/tutorials/dockervolumes/#/adding-a-data-volume>`__.
+  You can either let Docker manage the storage in the container (the easiest) or you can 
+  `mount a host directory as a data volume
+  <https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume>`.
+  However, if you mount a host directory, you must make sure that it exists and that
+  permissions are set correctly. You pass such a directory using ``-v
+  /my/own/datadir:/opt/my/own/datadir``, where mounts the ``/my/own/datadir`` from the
+  host system as ``/opt/cdap/sdk/my/own/datadir`` in the container.
+
+Controlling the CDAP Instance
+-----------------------------
+
+- To control the CDAP instance, use this command, substituting one of ``start``, ``restart``, ``stop``,
+  or ``status`` for ``<command>``:
+
+  .. tabbed-parsed-literal::
+    :tabs: "Linux or Mac OS X",Windows
+    :mapping: linux,windows
+    :dependent: linux-windows
+    :languages: console,shell-session
+    
+    .. Linux or Mac OS X
+    
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk <command>
+
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk start
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk restart
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
+
+    # To see the status:
+    $ docker exec -it cdap-standalone /opt/cdap/sdk/bin/cdap sdk status
+
+    .. Windows
+    
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk <command>
+
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk start
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk restart
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
+
+    # To see the status:
+    > docker exec -it cdap-standalone /opt/cdap/sdk/bin/cdap sdk status
 
-#. For a full list of Docker Commands, see the `Docker Command Line Documentation.
-   <https://docs.docker.com/reference/commandline/cli/>`__
+- When you are finished, stop CDAP and then shutdown the Docker machine:
+
+  .. tabbed-parsed-literal::
+    :tabs: "Linux or Mac OS X",Windows
+    :mapping: linux,windows
+    :dependent: linux-windows
+    :languages: console,shell-session
+    
+    .. Linux or Mac OS X
+    
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
+    $ docker-machine stop cdap
+
+    .. Windows
+    
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
```

Comment:
These should also be s/cdap-standalone/cdap-sdk/ and just remove the full PATH to `cdap sdk`